tests/broad/scala_test/src/main/scala/org/broadinstitute/dsp/pipelines/commandline/ConfigParser.scala

```diff
@@ -810,4 +810,22 @@ class ConfigParser
           )
         }
     )
+
+  note("")
+  germlineCloudPipelineCommandLineConfig(
+    VariantCalling, { config =>
+      (config.test, config.germlineCloudConfig.category) match {
+        case (VariantCalling, WorkflowTestCategory.Scientific) =>
+          failure(
```

Comment:
Should it just run the plumbing instead then?  (and not fail).

Reply:
I think that's dishonest to the user. If scientific tests are requested, I think the test suite should just say "Sorry Dave, I can't do that." and fail. I worry that making "scientific" tests run plumbing data might bite us in the future if we forget that its actually plumbing data and rely on it for something.

Reply:
Or maybe we should just run the scientific tests here then?  And not just say they're covered by another test.

Reply:
That would require extracting the intermediate BAM files from all the Scientific WGS and Exome samples and creating input files. It can be done, but I didn't really have time to do so before transitioning teams.

Reply:
Fair enough.  I'll bring it up with the team / make a ticket.